tasks/create_user.yml

```diff
@@ -2,6 +2,7 @@
 - name: "Create Gitea Group"
   group:
     name: "{{ gitea_group }}"
+    system: "yes"
```

Comment:
based on  [yamlint](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy) the recomended string for boolean values are `true` or `false`.

Reply:
Agreed, but the documentation says to use `yes` and `no`: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html

Just commited 81e2af0 that should solve this.